pkg/prom/metrics.go

```diff
@@ -0,0 +1,316 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prom
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	namespace = "onos"
```

Comment:
Is this k8s namespace? Can be anything not just onos? Or it is just used for internal purposes

Reply:
perhaps it is a default value

Reply:
This is a default value but it can be overwritten by the NewExporter, not necessarily related to k8s.
It can be any other string (e.g., riab, sdran, etc).